lib/index.js

```diff
@@ -1,14 +1,18 @@
 /*!
- * Copyright (c) 2020-2022 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2020-2024 Digital Bazaar, Inc. All rights reserved.
  */
+import * as credentialsV2Context from '@digitalbazaar/credentials-v2-context';
 import credentialsContext from 'credentials-context';
 import {jsonLdDocumentLoader} from '@bedrock/jsonld-document-loader';
 
 const contextModules = [
-  credentialsContext
+  credentialsContext,
+  credentialsV2Context,
```

Comment:
We need to figure out how to also export the constants from the credentials v2 package.